package.json

```diff
@@ -39,6 +39,7 @@
     "@storybook/addon-interactions": "^7.0.0-rc.6",
     "@storybook/addon-links": "^7.0.0-rc.6",
     "@storybook/addon-styling": "^0.3.2",
+    "@storybook/addon-viewport": "^6.5.16",
```

Comment:
Is this addon necessary?

By default storybook gives you a small mobile, large mobile, tablet, and wide screen viewports. I don't think this 
 is necessary.

Reply:
Thank you for feedback. I think we may need is in some case